data/reference/syntax.adoc

```diff
@@ -1,45 +1,58 @@
 = AsciiDoc Syntax
+:url-gem: http://rubygems.org/gems/asciidoctor
+:icons: font
 
 == Paragraphs
 
 A normal paragraph.
+Line breaks are not preserved.
 
-  An indented literal
-  paragraph.
+Use a blank line to separate paragraphs.
 
-== Formatted Text
+[%hardbreaks]
+This paragraph has the `hardbreaks` option.
+Notice that line breaks are now preserved.
 
-*bold text*.
-_emphasized text_.
-`monospaced text`.
+ A literal paragraph
+ disables text formatting
+ and preserves spaces and
+ line breaks.
 
-=== Unconstrained
+[sidebar#special.rolename]
```

Comment:
I don't know if it's clear that `special` is the id

Reply:
:+1: How about we use `id` and `role` to make it super clear.